magenta/models/rl_tuner/rl_tuner_ops.py

```diff
@@ -21,6 +21,7 @@
 
 import numpy as np
 import tensorflow as tf
+from tensorflow.contrib.rnn import rnn_cell
```

Comment:
to be consistent with the rest of our code, we'd prefer to just have one `import tensorflow as tf`, and then have the below references be things like `tf.contrib.rnn.LSTMCell`.